src/commands/moderation/simplecommand/simpleCommand.ts

```diff
@@ -0,0 +1,350 @@
+import { Colors, EmbedField, TextChannel } from 'discord.js';
+import moment from 'moment';
+import { CommandDefinition } from '../../../lib/command';
+import { Roles, Channels, CommandCategory } from '../../../constants';
+import { makeEmbed, makeLines } from '../../../lib/embed';
+import { getConn } from '../../../lib/db';
+import SimpleCommand from '../../../lib/schemas/simpleCommandSchema';
+
+const permittedRoles = [
+    Roles.ADMIN_TEAM,
+    Roles.MODERATION_TEAM,
+];
+
+const helpEmbed = (evokedCommand: String) => makeEmbed({
+    title: 'Simple Command - Help',
+    description: 'A command to manage simple commands that consist of a Title, Content and Color. This can be used to create temporary commands for different purposes.',
+    fields: [
+        {
+            name: 'Adding a Simple Command',
+            value: makeLines([
+                'To create a simple command, run the following bot command: ',
+                `\`${evokedCommand} add <command> <severity> <title> <content>\`.`,
+                '`command`: The command to be created that can be executed by regular users.',
+                '`severity`: The type can be `info`, `warning`, `warn`, `error` or `err`. Depending on the severity, the command output will have an appropriate color: regular, yellow or red.',
```

Comment:
As there are 3 severity types, could we keep `info`, `warning` and `error` only to keep it simple ish? 

No doubt this will need changing elsewhere!

Reply:
This is actually a mistake in the documentation where I forgot to clean up the warn and err, in the code it's not supported anymore :). Will get it cleaned up!